### PR TITLE
Made substantial Code Quality improvements

### LINKED ISSUE #463: Need to improve code quality
Going to make substantial improvements in code quality
All .html files are going to be updated for better code quality.

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #463 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-code quality of all HTML files
-

### CODE REVIEW

File: app/templates/404.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {% extends "layout.html" %}
 {% block body %}
 <div class="container">
-    <div class="row text-center">
-        <h1 class="not-found">404</h1>
-        <h1>Page Not Found</h1>
-    </div>
+	<div class="row text-center">
```
Comment (0x48piraj) [inline app/templates/404.html]: It was not needed, already indented.

File: app/templates/index.html [hunk 2]
```diff
@@ -1,128 +1,128 @@
...
-    <div class="row">
-        <form id="form1" action="{{ url_for('upload') }}" method="post" enctype="multipart/form-data" onsubmit="return validate()">
-            <div class="row">
-                <div class="container board">
-                    <div class="col-md-4 col-md-offset-4 col-sm-8 col-sm-offset-2 col-xs-10 col-xs-offset-1">
-                        <label>Choose your Data Source&emsp;<a id="icon" href="/guide" target="_blank"><i  class="fa fa-info-circle"></i></a></label>
-                        <ul style="list-style-type:none">
-                            <li>
-                                <input name="datasource" type="radio" id="csvupload"> Add CSV file</li>
+	<div class="row">
```
Comment (0x48piraj) [inline app/templates/index.html]: Again, not needed

File: app/templates/index.html [hunk 3]
```diff
@@ -1,128 +1,128 @@
...
-									<div class="btn-group">
-										<button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-											<span class="placeholder"> Select a background</span>
-											<span class="glyphicon glyphicon-chevron-down"></span>
-										</button>
-										<ul class="dropdown-menu">
-											{% for i in default_background %}
-											<li class="menu-options" data-item="{{i}}">
-												<img src="static/uploads/{{i}}"> {{i}}
+								<li>
```
Comment (0x48piraj) [inline app/templates/index.html]: Not needed

File: app/templates/index.html [hunk 4]
```diff
@@ -1,128 +1,128 @@
...
-						</ul>
-                        <button type="submit" disabled="disabled" class="btn btn-block submit-btn">Generate Badges</button>
-						<button type="button" disabled="disabled" class="btn btn-block btn-warning" id="preview-btn">Preview</button>
-                        <div id="preview" class="preview-image hidden">
-                        	<li class="preview-image-li" id="preview-li-1"></li>
-                        	<li class="preview-image-li" id="preview-li-2"></li>
-                        	<li class="preview-image-li" id="preview-li-3"></li>
-                        	<li class="preview-image-li" id="preview-li-4"></li>
-                        </div>
+										<div id="error" class="no-image-error"> Please select a default background or upload an image!</div>
```
Comment (0x48piraj) [inline app/templates/index.html]: Why adding multiple tabs and spaces which are already indented properly ?

**Comment by saarthakchats** [comment]
@0x48piraj I have just made it look neater and in order

**Comment by saarthakchats** [comment]
It is easier to understand now,

**Comment by 0x48piraj** [comment]
I still do not think and am not sure about your `re-indentation`
Please let maintainers decide :+1:

File: app/templates/404.html (hunk 1 above)
Comment (harshit98) [reply app/templates/404.html]: Yes @saarthakchats it was already indented.

**Pull request closed without merging**